images/prow-tests/install-kail.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
see this PR for reference: https://github.com/knative/test-infra/pull/1350

Reply:
Oh boy, more brittleness